reference-implementation/lib/abstract-ops/readable-streams.js

```diff
@@ -994,6 +995,18 @@ function ReadableByteStreamControllerEnqueue(controller, chunk) {
     return;
   }
 
+  if (controller._pendingPullIntos.length > 0) {
+    const firstPendingPullInto = controller._pendingPullIntos[0];
+    if (CanTransferArrayBuffer(firstPendingPullInto.buffer) === false) {
```

Comment:
We already know that the buffer of a pull-into descriptor has been transferred before when we constructed it, so this can be changed to `IsDetachedBuffer` too.